src/media/css/reviews.styl

```diff
@@ -33,7 +33,6 @@ select[name=rating] {
         font-size: 12px;
     }
     time {
-        position: absolute;
 
         +ltr() {
```

Comment:
I think without `absolute` the `right` and `left` properties below become meaningless.